app/reports/report_tasks.py

```diff
@@ -0,0 +1,29 @@
+from app import notify_celery
+from app.config import QueueNames
```

Comment:
Will these tasks run on all preview, staging and prod?

Any way you can restrict them so they use your config item.